islutils/builders.h

```diff
@@ -89,6 +89,30 @@ ScheduleNodeBuilder set(std::vector<ScheduleNodeBuilder> &&children);
 
 ScheduleNodeBuilder subtree(isl::schedule_node node);
 
+template <typename T>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t) {
```

Comment:
I would also add some static asserts or enable-ifs to check that `T` is either `ScheduleNodeBuilder` or an `std::vector` thereof.